### PR TITLE
feat: improve wsproxy address validation

### PR DESCRIPTION
**Pull request by agatha197**
resolves https://github.com/lablup/backend.ai-webui/issues/2597 

This pull request addresses a cosmetic issue by fixing a typo in the variable naming convention for resourceGroupWSProxyAddressInput. It also includes enhancements to the styling of the input fields by using the token value for the text label color in disabled and secondary states. Moreover, the validation pattern and auto-validation for wsproxy address input have been added to ensure correct URL format.

### How to test
1. go to resources - resource group tab
2. click the setting icon
3. check validation of the wsproxy address.

---

**Checklist:** (if applicable)

- [x] Mention to the original issue
- [ ] Documentation
- [ ] Minium required manager version
- [x] Specific setting for review (eg., KB link, endpoint or how to setup)
- [ ] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

**Comment by agatha197** [comment]
* **#2602** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2602?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @agatha197 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/components/backend-ai-resource-group-list.ts [hunk 1]
```diff
@@ -888,6 +888,10 @@ export default class BackendAIResourceGroupList extends BackendAIPage {
                   label="${_t('resourceGroup.WsproxyAddress')}"
                   placeholder="http://localhost:10200"
                   value="${this.resourceGroupInfo?.wsproxy_addr ?? ''}"
+                  pattern="^(https?://)[a-zA-Z0-9]([a-zA-Z0-9-]*[a-zA-Z0-9])*.[a-zA-Z]{2,5}(:[0-9]{1,5})?(/.*)?$"
```
Comment (ironAiken2) [inline src/components/backend-ai-resource-group-list.ts]: If set the type of mwc-textfield to url, won't it automatically validate the url type if I just provide the autoValidate attribute? @agatha197

Comment (agatha197) [reply src/components/backend-ai-resource-group-list.ts]: 😲

**Review by ironAiken2 — APPROVED** [review]
LGTM!

**Pull request merged**